device/rsvp/rsvpSessionAttribute.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Rsvp.SessionAttribute:
+      description: >-
+        Configuration for RSVP-TE SESSION_ATTRIBUTE object included in Path Messages as defined in RFC3209.
+        The bandwidth_protection_desired and node_protection_desired flags are defined in RFC4090 (Fast Reroute).
+      type: object
+      properties:
+        auto_generate_session_name:
+          description: >-
+            If this is enabled, an auto-generated Session Name is included in the SESSION_ATTRIBUTE object in the Path Message for this LSP.
+            It is suggested to include the Local IP, Remote IP, Tunnel ID and LSP ID in the auto-generated Session Name to ensure uniqueness of the name in the test.
+          type: boolean
+          default: true
+        session_name:
+          description: >-
+            If auto_generate_session_name is set to 'false', then the value of this field is used to fill the Session Name field.
+          type: string
+        setup_priority:
+          description: >-
+            Specifies the value of the Setup Priority field. This controls whether the LSP should pre-empt existing 
```

Comment:
When we are talking about the priority can we please specify about precedence - like 0 is the highest and 7 is the lowest? 
Same comment also applies for the field "holding_priority"